test/release/examples/primers/modules.chpl

```diff
@@ -319,9 +430,24 @@ module MainModule {
       writeln(other); // Outputs Conflict.other ('5.0 + 3.0i')
     }
 
+    /* ``import`` statements for unqualified access are somewhat similar to
+       ``only`` lists on ``use`` statements, though the syntax is different.
+       We saw how to ``import`` a single symbol for unqualified access earlier,
+       so this example demonstrates how to ``import`` multiple symbols for
+       unqualified access.
+    */
+    {
+      use modToUse;
+      import Conflict.{other, another};
+
+      writeln(foo); // Outputs modToUse.foo ('12')
+      writeln(bar); // Outputs modToUse.bar ('2')
+      writeln(other); // Outputs Conflict.other ('5.0 + 3.0i')
+    }
```

Comment:
I also find this to be more logically related to the general description of `import` with unqualified access rather than "limiting a use"

Reply:
That's fair.  I put it here for a few reasons:
- it is logically similar to `only` lists
- the section where import statements were introduced was getting a little too long and separating `use` information from `import` information
- and because the renaming portion is relevant to both `only` lists and this syntax